jenkins-pipelines/features-scale-450-500-cluster-test.jenkinsfile

```diff
@@ -8,7 +8,7 @@ longevityPipeline(
     region: 'eu-west-1',
     post_behavior_db_nodes: 'destroy',
     test_name: 'grow_cluster_test.GrowClusterTest.test_grow_x_to_y',
-    test_config: '''["test-cases/features/scale-cluster.yaml", "configurations/scale-450-500.yaml"]''',
+    test_config: '''["test-cases/scale/scale-cluster.yaml", "configurations/scale-450-500.yaml"]''',
 
```

Comment:
Maybe also in that he configuration folder you want a scale folder ?

Reply:
yes, configurations are quite organized. done. thanks.